testssl.sh

```diff
@@ -8805,22 +8805,29 @@ run_drown() {
+     local cbc_cipher_list="ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:SRP-DSS-AES-256-CBC-SHA:SRP-RSA-AES-256-CBC-SHA:SRP-AES-256-CBC-SHA:DHE-PSK-AES256-CBC-SHA:DHE-RSA-AES256-SHA:DHE-DSS-AES256-SHA:DH-RSA-AES256-SHA:DH-DSS-AES256-SHA:DHE-RSA-CAMELLIA256-SHA:DHE-DSS-CAMELLIA256-SHA:DH-RSA-CAMELLIA256-SHA:DH-DSS-CAMELLIA256-SHA:AECDH-AES256-SHA:ADH-AES256-SHA:ADH-CAMELLIA256-SHA:ECDH-RSA-AES256-SHA:ECDH-ECDSA-AES256-SHA:AES256-SHA:ECDHE-PSK-AES256-CBC-SHA:CAMELLIA256-SHA:RSA-PSK-AES256-CBC-SHA:PSK-AES256-CBC-SHA:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:SRP-DSS-AES-128-CBC-SHA:SRP-RSA-AES-128-CBC-SHA:SRP-AES-128-CBC-SHA:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA:DH-RSA-AES128-SHA:DH-DSS-AES128-SHA:DHE-RSA-SEED-SHA:DHE-DSS-SEED-SHA:DH-RSA-SEED-SHA:DH-DSS-SEED-SHA:DHE-RSA-CAMELLIA128-SHA:DHE-DSS-CAMELLIA128-SHA:DH-RSA-CAMELLIA128-SHA:DH-DSS-CAMELLIA128-SHA:AECDH-AES128-SHA:ADH-AES128-SHA:ADH-SEED-SHA:ADH-CAMELLIA128-SHA:ECDH-RSA-AES128-SHA:ECDH-ECDSA-AES128-SHA:AES128-SHA:ECDHE-PSK-AES128-CBC-SHA:DHE-PSK-AES128-CBC-SHA:SEED-SHA:CAMELLIA128-SHA:IDEA-CBC-SHA:RSA-PSK-AES128-CBC-SHA:PSK-AES128-CBC-SHA:KRB5-IDEA-CBC-SHA:KRB5-IDEA-CBC-MD5:ECDHE-RSA-DES-CBC3-SHA:ECDHE-ECDSA-DES-CBC3-SHA:SRP-DSS-3DES-EDE-CBC-SHA:SRP-RSA-3DES-EDE-CBC-SHA:SRP-3DES-EDE-CBC-SHA:EDH-RSA-DES-CBC3-SHA:EDH-DSS-DES-CBC3-SHA:DH-RSA-DES-CBC3-SHA:DH-DSS-DES-CBC3-SHA:AECDH-DES-CBC3-SHA:ADH-DES-CBC3-SHA:ECDH-RSA-DES-CBC3-SHA:ECDH-ECDSA-DES-CBC3-SHA:DES-CBC3-SHA:RSA-PSK-3DES-EDE-CBC-SHA:PSK-3DES-EDE-CBC-SHA:KRB5-DES-CBC3-SHA:KRB5-DES-CBC3-MD5:ECDHE-PSK-3DES-EDE-CBC-SHA:DHE-PSK-3DES-EDE-CBC-SHA:EXP1024-DHE-DSS-DES-CBC-SHA:EDH-RSA-DES-CBC-SHA:EDH-DSS-DES-CBC-SHA:DH-RSA-DES-CBC-SHA:DH-DSS-DES-CBC-SHA:ADH-DES-CBC-SHA:EXP1024-DES-CBC-SHA:DES-CBC-SHA:KRB5-DES-CBC-SHA:KRB5-DES-CBC-MD5:EXP-EDH-RSA-DES-CBC-SHA:EXP-EDH-DSS-DES-CBC-SHA:EXP-ADH-DES-CBC-SHA:EXP-DES-CBC-SHA:EXP-RC2-CBC-MD5:EXP-KRB5-RC2-CBC-SHA:EXP-KRB5-DES-CBC-SHA:EXP-KRB5-RC2-CBC-MD5:EXP-KRB5-DES-CBC-MD5:EXP-DH-DSS-DES-CBC-SHA:EXP-DH-RSA-DES-CBC-SHA"
```

Comment:
As I was working on this code, at one point I included a line in the `for (( i=0; i < TLS_NR_CIPHERS; i++ )); do` loop that created `cbc_cipher_list` on-the-fly, and the new list is what was created. I compared the old and the new list, and they differed as shown below.

It seemed reasonable to add the PSK ciphers, since the old `cbc_cipher_list` already included PSK ciphers.

Most of the ciphers that were removed were either "-SHA256" or "-SHA384" ciphers, and they don't need to be in the list, since such ciphers only work with TLSv1.2 and `cbc_cipher_list` is only tested against SSLv3 and TLSv1.0. According to OpenSSL the other ciphers are SSLv2 ciphers, so they don't belong on the list either:
```
> openssl.Linux.x86_64.krb5 ciphers -V DES-CBC3-MD5:DES-CBC-MD5:EXP-RC2-CBC-MD5:IDEA-CBC-MD5:RC2-CBC-MD5
     0x07,0x00,0xC0 - DES-CBC3-MD5            SSLv2 Kx=RSA      Au=RSA  Enc=3DES(168) Mac=MD5 
     0x06,0x00,0x40 - DES-CBC-MD5             SSLv2 Kx=RSA      Au=RSA  Enc=DES(56)   Mac=MD5 
          0x00,0x06 - EXP-RC2-CBC-MD5         SSLv3 Kx=RSA(512) Au=RSA  Enc=RC2(40)   Mac=MD5  export
     0x04,0x00,0x80 - EXP-RC2-CBC-MD5         SSLv2 Kx=RSA(512) Au=RSA  Enc=RC2(40)   Mac=MD5  export
     0x05,0x00,0x80 - IDEA-CBC-MD5            SSLv2 Kx=RSA      Au=RSA  Enc=IDEA(128) Mac=MD5 
     0x03,0x00,0x80 - RC2-CBC-MD5             SSLv2 Kx=RSA      Au=RSA  Enc=RC2(128)  Mac=MD5
```
"EXP-RC2-CBC-MD5" is an exception, in that in that it is both an SSLv2 cipher and an SSLv3 cipher. However, it appeared twice in the old list; now it appears just once.

----------------------------------------------------------------

```
added:
 DHE-PSK-3DES-EDE-CBC-SHA
 DHE-PSK-AES128-CBC-SHA
 DHE-PSK-AES256-CBC-SHA
 ECDHE-PSK-3DES-EDE-CBC-SHA
 ECDHE-PSK-AES128-CBC-SHA
 ECDHE-PSK-AES256-CBC-SHA
 RSA-PSK-3DES-EDE-CBC-SHA
 RSA-PSK-AES128-CBC-SHA
 RSA-PSK-AES256-CBC-SHA

removed:
 ADH-AES128-SHA256
 ADH-AES256-SHA256
 AES128-SHA256
 AES256-SHA256
 DES-CBC3-MD5 
 DES-CBC-MD5
 DH-DSS-AES128-SHA256
 DH-DSS-AES256-SHA256
 DHE-DSS-AES128-SHA256
 DHE-DSS-AES256-SHA256
 DHE-RSA-AES128-SHA256
 DHE-RSA-AES256-SHA256
 DH-RSA-AES128-SHA256
 DH-RSA-AES256-SHA256
 ECDH-ECDSA-AES128-SHA256
 ECDH-ECDSA-AES256-SHA384
 ECDHE-ECDSA-AES128-SHA256
 ECDHE-ECDSA-AES256-SHA384
 ECDHE-RSA-AES128-SHA256
 ECDHE-RSA-AES256-SHA384
 ECDH-RSA-AES128-SHA256
 ECDH-RSA-AES256-SHA384
 EXP-RC2-CBC-MD5
 IDEA-CBC-MD5
 RC2-CBC-MD